ui/src/app/applications/components/application-sync-panel/application-sync-panel.tsx

```diff
@@ -7,10 +7,11 @@ import * as models from '../../../shared/models';
 import { services } from '../../../shared/services';
 import { ComparisonStatusIcon, nodeKey } from '../utils';
 
-export const ApplicationSyncPanel = ({application, selectedResource, hide}: {
+export const ApplicationSyncPanel = ({application, selectedResource, hide, update}: {
     application: models.Application,
     selectedResource: string,
     hide: () => any;
+    update: (ignoreWarnings: boolean) => void;
```

Comment:
probably could be named`updateIgnoreWarnings`?

Reply:
could be optional, i.e. `updateIgnoreWarnings?:`

Reply:
Good point. Done.